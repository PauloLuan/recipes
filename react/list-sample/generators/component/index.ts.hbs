export * from './{{pascalCase componentName}}.component'
