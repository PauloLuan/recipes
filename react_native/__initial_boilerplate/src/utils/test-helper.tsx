import { render, RenderAPI } from '@testing-library/react-native'
import React from 'react'
import { ThemeProvider } from 'styled-components/native'
import theme from '../global/styles/theme'

export const renderWithTheme = (children: React.ReactNode): RenderAPI =>
  render(<ThemeProvider theme={theme}>{children}</ThemeProvider>)
