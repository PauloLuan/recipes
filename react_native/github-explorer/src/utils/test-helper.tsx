import { render, RenderAPI } from '@testing-library/react-native'
import React from 'react'
import { ThemeProvider } from 'styled-components/native'
import theme from '../global/styles/theme'

export const renderWithTheme = (children: React.ReactNode): RenderAPI =>
  render(<ThemeProvider theme={theme}>{children}</ThemeProvider>)

export const mockNavigation = () => {
  const mockedNavigate = jest.fn()

  jest.mock('@react-navigation/native', () => {
    const actualNav = jest.requireActual('@react-navigation/native')
    return {
      ...actualNav,
      useNavigation: () => ({
        navigate: mockedNavigate
      })
    }
  })

  return { mockedNavigate }
}
