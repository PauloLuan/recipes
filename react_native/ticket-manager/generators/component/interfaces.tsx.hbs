import { I{{capitalize componentName}}Props } from 'native-base'

export interface {{capitalize componentName}}Props extends I{{capitalize componentName}}Props {
  testId?: string
}
