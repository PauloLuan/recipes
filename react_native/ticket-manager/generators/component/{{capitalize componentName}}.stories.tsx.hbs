import { storiesOf } from '@storybook/react-native'
import React from 'react'
import { {{capitalize componentName}} } from './{{capitalize componentName}}'

storiesOf('Components/{{capitalize componentName}}', module).add('default: ', () => <{{capitalize componentName}} />)
