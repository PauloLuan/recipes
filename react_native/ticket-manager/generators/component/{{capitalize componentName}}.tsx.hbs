import React from 'react'
import { I{{capitalize componentName}}Props, {{capitalize componentName}} as NativeBase{{capitalize componentName}} } from 'native-base'

export interface {{capitalize componentName}}Props extends I{{capitalize componentName}}Props {
  testId?: string
}

export const {{capitalize componentName}} = ({
  testId = '{{kebabCase componentName}}-id',
  ...props
}: {{capitalize componentName}}Props) => {
  return (
    <NativeBase{{capitalize componentName}} testID={testId} {...props}></NativeBase{{capitalize componentName}}>
  )
}
