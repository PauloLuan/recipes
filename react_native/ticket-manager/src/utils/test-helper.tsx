import { render, RenderAPI } from '@testing-library/react-native'
import { NativeBaseProvider } from 'native-base'
import React from 'react'
import { theme } from '../global/styles/theme'

export const renderWithTheme = (children: React.ReactNode): RenderAPI =>
  render(<NativeBaseProvider theme={theme}>{children}</NativeBaseProvider>)
