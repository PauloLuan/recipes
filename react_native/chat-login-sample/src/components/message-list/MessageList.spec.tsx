import { waitFor } from '@testing-library/react-native'
import React from 'react'
import { renderWithTheme } from '../../utils/test-helper'
import { MessageList } from './MessageList.component'

describe('<MessageList />', () => {
  const testId = 'component-message-list-id'

  it('should render the MessageList component', async () => {
    const { queryByTestId, toJSON } = renderWithTheme(
      <MessageList testId={testId} />
    )
    await waitFor(() => expect(queryByTestId(testId)).toBeTruthy())
    expect(toJSON()).toMatchSnapshot()
  })
})
