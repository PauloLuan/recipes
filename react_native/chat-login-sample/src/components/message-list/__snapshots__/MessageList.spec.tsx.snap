// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MessageList /> should render the MessageList component 1`] = `
<RCTScrollView
  style={
    Array [
      Object {},
    ]
  }
  testID="component-message-list-id"
>
  <View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "black",
            "flexDirection": "row",
            "height": 100,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          },
        ]
      }
      testID="component-Message-id"
    >
      <ViewManagerAdapter_ExpoLinearGradient
        proxiedProperties={
          Object {
            "avatarSize": 28,
            "borderRadius": 16,
            "colors": Array [
              4279994175,
              4294954270,
            ],
            "containerSize": 32,
            "endPoint": undefined,
            "locations": undefined,
            "startPoint": undefined,
          }
        }
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 16,
              "height": 32,
              "justifyContent": "center",
              "width": 32,
            },
          ]
        }
      >
        <Image
          avatarSize={28}
          borderRadius={16}
          source={
            Object {
              "uri": "file://assets/full/path/to/directory/name.type",
            }
          }
          style={
            Array [
              Object {
                "borderRadius": 16,
                "height": 28,
                "width": 28,
              },
            ]
          }
          testID="component-user-photo-id"
        />
      </ViewManagerAdapter_ExpoLinearGradient>
      <View
        style={
          Array [
            Object {
              "flexDirection": "column",
              "marginLeft": 10,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontSize": 10,
              },
            ]
          }
        >
          Paulo Luan
        </Text>
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
                "flexWrap": "wrap",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 20,
                "marginTop": 10,
              },
            ]
          }
        >
          <Text
            numberOfLines={2}
            style={
              Array [
                Object {
                  "color": "white",
                  "flexWrap": "wrap",
                  "fontSize": 14,
                },
              ]
            }
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "black",
            "flexDirection": "row",
            "height": 100,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          },
        ]
      }
      testID="component-Message-id"
    >
      <ViewManagerAdapter_ExpoLinearGradient
        proxiedProperties={
          Object {
            "avatarSize": 28,
            "borderRadius": 16,
            "colors": Array [
              4279994175,
              4294954270,
            ],
            "containerSize": 32,
            "endPoint": undefined,
            "locations": undefined,
            "startPoint": undefined,
          }
        }
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 16,
              "height": 32,
              "justifyContent": "center",
              "width": 32,
            },
          ]
        }
      >
        <Image
          avatarSize={28}
          borderRadius={16}
          source={
            Object {
              "uri": "file://assets/full/path/to/directory/name.type",
            }
          }
          style={
            Array [
              Object {
                "borderRadius": 16,
                "height": 28,
                "width": 28,
              },
            ]
          }
          testID="component-user-photo-id"
        />
      </ViewManagerAdapter_ExpoLinearGradient>
      <View
        style={
          Array [
            Object {
              "flexDirection": "column",
              "marginLeft": 10,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontSize": 10,
              },
            ]
          }
        >
          Paulo Luan
        </Text>
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
                "flexWrap": "wrap",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 20,
                "marginTop": 10,
              },
            ]
          }
        >
          <Text
            numberOfLines={2}
            style={
              Array [
                Object {
                  "color": "white",
                  "flexWrap": "wrap",
                  "fontSize": 14,
                },
              ]
            }
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "black",
            "flexDirection": "row",
            "height": 100,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          },
        ]
      }
      testID="component-Message-id"
    >
      <ViewManagerAdapter_ExpoLinearGradient
        proxiedProperties={
          Object {
            "avatarSize": 28,
            "borderRadius": 16,
            "colors": Array [
              4279994175,
              4294954270,
            ],
            "containerSize": 32,
            "endPoint": undefined,
            "locations": undefined,
            "startPoint": undefined,
          }
        }
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 16,
              "height": 32,
              "justifyContent": "center",
              "width": 32,
            },
          ]
        }
      >
        <Image
          avatarSize={28}
          borderRadius={16}
          source={
            Object {
              "uri": "file://assets/full/path/to/directory/name.type",
            }
          }
          style={
            Array [
              Object {
                "borderRadius": 16,
                "height": 28,
                "width": 28,
              },
            ]
          }
          testID="component-user-photo-id"
        />
      </ViewManagerAdapter_ExpoLinearGradient>
      <View
        style={
          Array [
            Object {
              "flexDirection": "column",
              "marginLeft": 10,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontSize": 10,
              },
            ]
          }
        >
          Paulo Luan
        </Text>
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
                "flexWrap": "wrap",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 20,
                "marginTop": 10,
              },
            ]
          }
        >
          <Text
            numberOfLines={2}
            style={
              Array [
                Object {
                  "color": "white",
                  "flexWrap": "wrap",
                  "fontSize": 14,
                },
              ]
            }
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "black",
            "flexDirection": "row",
            "height": 100,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          },
        ]
      }
      testID="component-Message-id"
    >
      <ViewManagerAdapter_ExpoLinearGradient
        proxiedProperties={
          Object {
            "avatarSize": 28,
            "borderRadius": 16,
            "colors": Array [
              4279994175,
              4294954270,
            ],
            "containerSize": 32,
            "endPoint": undefined,
            "locations": undefined,
            "startPoint": undefined,
          }
        }
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 16,
              "height": 32,
              "justifyContent": "center",
              "width": 32,
            },
          ]
        }
      >
        <Image
          avatarSize={28}
          borderRadius={16}
          source={
            Object {
              "uri": "file://assets/full/path/to/directory/name.type",
            }
          }
          style={
            Array [
              Object {
                "borderRadius": 16,
                "height": 28,
                "width": 28,
              },
            ]
          }
          testID="component-user-photo-id"
        />
      </ViewManagerAdapter_ExpoLinearGradient>
      <View
        style={
          Array [
            Object {
              "flexDirection": "column",
              "marginLeft": 10,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontSize": 10,
              },
            ]
          }
        >
          Paulo Luan
        </Text>
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
                "flexWrap": "wrap",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 20,
                "marginTop": 10,
              },
            ]
          }
        >
          <Text
            numberOfLines={2}
            style={
              Array [
                Object {
                  "color": "white",
                  "flexWrap": "wrap",
                  "fontSize": 14,
                },
              ]
            }
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "black",
            "flexDirection": "row",
            "height": 100,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          },
        ]
      }
      testID="component-Message-id"
    >
      <ViewManagerAdapter_ExpoLinearGradient
        proxiedProperties={
          Object {
            "avatarSize": 28,
            "borderRadius": 16,
            "colors": Array [
              4279994175,
              4294954270,
            ],
            "containerSize": 32,
            "endPoint": undefined,
            "locations": undefined,
            "startPoint": undefined,
          }
        }
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 16,
              "height": 32,
              "justifyContent": "center",
              "width": 32,
            },
          ]
        }
      >
        <Image
          avatarSize={28}
          borderRadius={16}
          source={
            Object {
              "uri": "file://assets/full/path/to/directory/name.type",
            }
          }
          style={
            Array [
              Object {
                "borderRadius": 16,
                "height": 28,
                "width": 28,
              },
            ]
          }
          testID="component-user-photo-id"
        />
      </ViewManagerAdapter_ExpoLinearGradient>
      <View
        style={
          Array [
            Object {
              "flexDirection": "column",
              "marginLeft": 10,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontSize": 10,
              },
            ]
          }
        >
          Paulo Luan
        </Text>
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
                "flexWrap": "wrap",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 20,
                "marginTop": 10,
              },
            ]
          }
        >
          <Text
            numberOfLines={2}
            style={
              Array [
                Object {
                  "color": "white",
                  "flexWrap": "wrap",
                  "fontSize": 14,
                },
              ]
            }
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "black",
            "flexDirection": "row",
            "height": 100,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          },
        ]
      }
      testID="component-Message-id"
    >
      <ViewManagerAdapter_ExpoLinearGradient
        proxiedProperties={
          Object {
            "avatarSize": 28,
            "borderRadius": 16,
            "colors": Array [
              4279994175,
              4294954270,
            ],
            "containerSize": 32,
            "endPoint": undefined,
            "locations": undefined,
            "startPoint": undefined,
          }
        }
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 16,
              "height": 32,
              "justifyContent": "center",
              "width": 32,
            },
          ]
        }
      >
        <Image
          avatarSize={28}
          borderRadius={16}
          source={
            Object {
              "uri": "file://assets/full/path/to/directory/name.type",
            }
          }
          style={
            Array [
              Object {
                "borderRadius": 16,
                "height": 28,
                "width": 28,
              },
            ]
          }
          testID="component-user-photo-id"
        />
      </ViewManagerAdapter_ExpoLinearGradient>
      <View
        style={
          Array [
            Object {
              "flexDirection": "column",
              "marginLeft": 10,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontSize": 10,
              },
            ]
          }
        >
          Paulo Luan
        </Text>
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
                "flexWrap": "wrap",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 20,
                "marginTop": 10,
              },
            ]
          }
        >
          <Text
            numberOfLines={2}
            style={
              Array [
                Object {
                  "color": "white",
                  "flexWrap": "wrap",
                  "fontSize": 14,
                },
              ]
            }
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "black",
            "flexDirection": "row",
            "height": 100,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          },
        ]
      }
      testID="component-Message-id"
    >
      <ViewManagerAdapter_ExpoLinearGradient
        proxiedProperties={
          Object {
            "avatarSize": 28,
            "borderRadius": 16,
            "colors": Array [
              4279994175,
              4294954270,
            ],
            "containerSize": 32,
            "endPoint": undefined,
            "locations": undefined,
            "startPoint": undefined,
          }
        }
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 16,
              "height": 32,
              "justifyContent": "center",
              "width": 32,
            },
          ]
        }
      >
        <Image
          avatarSize={28}
          borderRadius={16}
          source={
            Object {
              "uri": "file://assets/full/path/to/directory/name.type",
            }
          }
          style={
            Array [
              Object {
                "borderRadius": 16,
                "height": 28,
                "width": 28,
              },
            ]
          }
          testID="component-user-photo-id"
        />
      </ViewManagerAdapter_ExpoLinearGradient>
      <View
        style={
          Array [
            Object {
              "flexDirection": "column",
              "marginLeft": 10,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontSize": 10,
              },
            ]
          }
        >
          Paulo Luan
        </Text>
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
                "flexWrap": "wrap",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 20,
                "marginTop": 10,
              },
            ]
          }
        >
          <Text
            numberOfLines={2}
            style={
              Array [
                Object {
                  "color": "white",
                  "flexWrap": "wrap",
                  "fontSize": 14,
                },
              ]
            }
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "black",
            "flexDirection": "row",
            "height": 100,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          },
        ]
      }
      testID="component-Message-id"
    >
      <ViewManagerAdapter_ExpoLinearGradient
        proxiedProperties={
          Object {
            "avatarSize": 28,
            "borderRadius": 16,
            "colors": Array [
              4279994175,
              4294954270,
            ],
            "containerSize": 32,
            "endPoint": undefined,
            "locations": undefined,
            "startPoint": undefined,
          }
        }
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 16,
              "height": 32,
              "justifyContent": "center",
              "width": 32,
            },
          ]
        }
      >
        <Image
          avatarSize={28}
          borderRadius={16}
          source={
            Object {
              "uri": "file://assets/full/path/to/directory/name.type",
            }
          }
          style={
            Array [
              Object {
                "borderRadius": 16,
                "height": 28,
                "width": 28,
              },
            ]
          }
          testID="component-user-photo-id"
        />
      </ViewManagerAdapter_ExpoLinearGradient>
      <View
        style={
          Array [
            Object {
              "flexDirection": "column",
              "marginLeft": 10,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontSize": 10,
              },
            ]
          }
        >
          Paulo Luan
        </Text>
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
                "flexWrap": "wrap",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 20,
                "marginTop": 10,
              },
            ]
          }
        >
          <Text
            numberOfLines={2}
            style={
              Array [
                Object {
                  "color": "white",
                  "flexWrap": "wrap",
                  "fontSize": 14,
                },
              ]
            }
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </Text>
        </View>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "black",
            "flexDirection": "row",
            "height": 100,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          },
        ]
      }
      testID="component-Message-id"
    >
      <ViewManagerAdapter_ExpoLinearGradient
        proxiedProperties={
          Object {
            "avatarSize": 28,
            "borderRadius": 16,
            "colors": Array [
              4279994175,
              4294954270,
            ],
            "containerSize": 32,
            "endPoint": undefined,
            "locations": undefined,
            "startPoint": undefined,
          }
        }
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 16,
              "height": 32,
              "justifyContent": "center",
              "width": 32,
            },
          ]
        }
      >
        <Image
          avatarSize={28}
          borderRadius={16}
          source={
            Object {
              "uri": "file://assets/full/path/to/directory/name.type",
            }
          }
          style={
            Array [
              Object {
                "borderRadius": 16,
                "height": 28,
                "width": 28,
              },
            ]
          }
          testID="component-user-photo-id"
        />
      </ViewManagerAdapter_ExpoLinearGradient>
      <View
        style={
          Array [
            Object {
              "flexDirection": "column",
              "marginLeft": 10,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontSize": 10,
              },
            ]
          }
        >
          Paulo Luan
        </Text>
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
                "flexWrap": "wrap",
                "marginBottom": 0,
                "marginLeft": 0,
                "marginRight": 20,
                "marginTop": 10,
              },
            ]
          }
        >
          <Text
            numberOfLines={2}
            style={
              Array [
                Object {
                  "color": "white",
                  "flexWrap": "wrap",
                  "fontSize": 14,
                },
              ]
            }
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
          </Text>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
